client/apps/edit/components/content2/sections/header/controls.jsx

```diff
@@ -22,15 +27,62 @@ export default class FeatureHeaderControls extends Component {
 
   getMenuColor = () => {
     const { hero } = this.props
+
     if (hero && hero.type === 'fullscreen' && hero.url && hero.url.length) {
       return 'white'
     } else {
       return 'black'
     }
   }
 
+  toggleLayoutControls = () => {
+    const { isLayoutOpen, isVideoOpen } = this.state
+
+    this.setState({
+      isLayoutOpen: !isLayoutOpen
+    })
+
+    if (isVideoOpen) {
+      this.toggleVideoControls() // hide
+    }
+
+  }
+
+  toggleVideoControls = () => {
+    const { isLayoutOpen, isVideoOpen } = this.state
+
+    this.setState({
+      isVideoOpen: !isVideoOpen
+    })
+
+    if (isLayoutOpen) {
+      this.toggleLayoutControls()
+    }
+  }
+
+  renderVideoEmbed () {
+    const { article } = this.props
+
+    if (this.state.isVideoOpen) {
+      return (
```

Comment:
Since we already have the preview rendering in the main layout, you could probably use just the video controls component here.